packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -117,6 +118,15 @@ static Object toJson(CameraPosition position) {
     return data;
   }
 
+  static Object toJson(MarkerId markerId) {
```

Comment:
Don't change it as part of this PR to stay consistent, but all of these overloads of function with the same name feels error prone to me, I'd consider given each of them a different name, Also it's not converting to JSON but to a map.

Reply:
filed: https://github.com/flutter/flutter/issues/28363